lib/allscripts_api/client.rb

```diff
@@ -5,19 +5,20 @@
 module AllscriptsApi
   # Client serves as an entry point for making calls
   class Client
+    include AllscriptsApi::NamedMagicMethods
     attr_reader :adapter, :unity_url, :app_name, :token
     ENDPOINT = "/Unity/UnityService.svc/json".freeze
 
     # Instantiation of the Client
     #
     # @param app_name [String] app name assigned by Allscripts
     # @param url [String] Allscripts URL to be used to make Unity API calls
-    # @param username [String] the applications username supplied by Allscripts
-    # @param password [String] the applications password supplied by Allscripts
-    def initialize(app_name, url, username, password)
+    # @param app_username [String] the app username supplied by Allscripts
+    # @param app_password [String] the app password supplied by Allscripts
+    def initialize(app_name, url, app_username, app_password)
```

Comment:
I'd shuffle the order so that `url` is first because that's the only "dynamic" piece at the app level. It'd be easier if we end up wrapping this inside of an app to only require the `url`.

Reply:
good call.